.github/workflows/examples-java-push.yml

```diff
@@ -0,0 +1,43 @@
+name: Build `examples/java` pushes
+
+on:
+  push:
+    paths:
+      - "examples/java/**"
+      - ".github/workflows/examples-java-pr.yml"
```

Comment:
should this be examples-java-push.yml? Also not sure if this is needed here

Reply:
Good spot :+1:

Not needed because it shouldn't be required for `push`, or ..?

Reply:
I don't think the path to the yml file is required here, I think `paths` is just for including/excluding files you want the workflow to run on

Reply:
Ah gotcha, so the idea of this is that (from what I understood from reading through a few places), because we're treating this as a monorepo, and only want to build the project when it's updated, we need to make sure that either code changes _or_ changes to the Workflows trigger the Workflows, because otherwise I'd be able to make a change to the workflow and it wouldn't trigger itself

Reply:
ah ok, that makes sense